src/components/SecondaryPanes/Frames.js

```diff
@@ -55,6 +55,14 @@ const Frames = createClass({
       || showAllFrames !== nextState.showAllFrames;
   },
 
+  componentWillMount() {
+    document.addEventListener("keyup", this.onKeyUp, false);
+  },
+
+  componentWillUnmount() {
+    document.removeEventListener("keyup", this.onKeyUp, false);
```

Comment:
But if I will move this, it will not listen keyUp's, until we click on some frame.
It will work fine: "click on some frame, then hover another and press Enter", and in this case "hover some frame and press Enter", onKeyUp event will not be fire for component.